Week-1/day1/solution.txt

```diff
@@ -0,0 +1,83 @@
+Q. What is a protocol stack, and how is it used in web development?
+Ans: A protocol stack is a layered system of communication protocols used to transmit data over a network. In web development, it is used to transmit data between the web server and the web browser. In short, it is a system of protocols that allows web applications to communicate with each other reliably. The most common protocol stack used in web development is the TCP/IP stack. The TCP/IP stack consists of four layers:
```

Comment:
Consider adding examples such as how DNS works within the Application layer or how the TCP protocol manages packet retransmission. May be a graphic will help